recipes/xarray-einstats/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Generated using grayskull; edited by @ericmjl.
+{% set name = "xarray-einstats" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xarray-einstats-{{ version }}.tar.gz
+  sha256: 5c4e00d09172daf665cbd5e533a897896b84f37feb5cab0fd75829477fbcda7d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip >=21.3
+    - python
```

Comment:
```suggestion
    - python >=3.7
    - flit
```

Reply:
the azure jobs are complaining about flit, it is a build system dependency in pyproject: https://github.com/arviz-devs/xarray-einstats/blob/main/pyproject.toml#L2. I think it needs to be here like it is in the poliastro recipe for example: https://github.com/conda-forge/poliastro-feedstock/blob/main/recipe/meta.yaml

python could be bigger than 3.6 as the linter suggests, but the run is 3.7 so it is probably easier and less prone to confusion to use 3.7 in both

Reply:
Thanks for the catch, @OriolAbril! Okie just made the changes.